packages/vscode-ui5-language-assistant/test/unit/.mocharc.js

```diff
@@ -1,3 +1,7 @@
+const chai = require("chai");
+const deepEqualInAnyOrder = require("deep-equal-in-any-order");
+chai.use(deepEqualInAnyOrder);
```

Comment:
Should this really be in the configuration file? Wouldn't it be better to do it in the test file?

Reply:
Its a **plugin** that effects the chai **global** state.
If I put it in each test file it would only have the desired side effect the first time it is invoked.
And we would have to track which test use/does not use/no longer uses these plugin's assertions.

Yeah we only have 1-2 test files here, so it does not really matter, but I think its a better practice to manage the Chai plugins "**centrally**".

Reply:
The reason it bothers me is that you are changing the state of chai by importing (reading) the mocha configuration file. It doesn't sound like it's an operation that should have side effects. I would prefer to add the `chai.use` line in every test file that uses these plugins.
But if you think this is the better approach I won't prevent merging it, it's not that critical.